README.md

```diff
@@ -201,6 +203,7 @@ The supported rules are described below:
 | parameter_order              | Flag any operations with optional parameters before a required param.               | shared   |
 | no_request_body_content      | [Flag any operations with a `requestBody` that does not have a `content` field.][3] | oas3     |
 | no_request_body_name         | Flag any operations with a non-form `requestBody` that does not have a name set with `x-codegen-request-body-name`. | oas3|
+| no_response_body             | Flag any non-204 success responses without a response body.                         | oas3     |
```

Comment:
You have the README descriptions in the `operations` section but the default in the `responses` section. I think it should be in the `responses` section, so you just need to move the README descriptions